2022-02-12 20:36:22,234 INFO    MainThread:4379 [wandb_setup.py:_flush():69] setting env: {'project': 'telco_churn', 'run_group': 'development'}
2022-02-12 20:36:22,234 INFO    MainThread:4379 [wandb_setup.py:_flush():69] setting login settings: {}
2022-02-12 20:36:22,234 INFO    MainThread:4379 [wandb_init.py:_log_setup():336] Logging user logs to /Users/jelaleddin/MLPipeline-TelcoCustomerChurn/components/train_val_test_split/wandb/run-20220212_203622-1g5mvyuv/logs/debug.log
2022-02-12 20:36:22,235 INFO    MainThread:4379 [wandb_init.py:_log_setup():337] Logging internal logs to /Users/jelaleddin/MLPipeline-TelcoCustomerChurn/components/train_val_test_split/wandb/run-20220212_203622-1g5mvyuv/logs/debug-internal.log
2022-02-12 20:36:22,235 INFO    MainThread:4379 [wandb_init.py:init():369] calling init triggers
2022-02-12 20:36:22,235 INFO    MainThread:4379 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2022-02-12 20:36:22,235 INFO    MainThread:4379 [wandb_init.py:init():418] starting backend
2022-02-12 20:36:22,235 INFO    MainThread:4379 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-02-12 20:36:22,247 INFO    MainThread:4379 [backend.py:ensure_launched():132] starting backend process...
2022-02-12 20:36:22,257 INFO    MainThread:4379 [backend.py:ensure_launched():136] started backend process with pid: 4386
2022-02-12 20:36:22,260 INFO    MainThread:4379 [wandb_init.py:init():423] backend started and connected
2022-02-12 20:36:22,264 INFO    MainThread:4379 [wandb_init.py:init():465] updated telemetry
2022-02-12 20:36:22,271 INFO    MainThread:4379 [wandb_init.py:init():484] communicating current version
2022-02-12 20:36:27,278 INFO    MainThread:4379 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2022-02-12 20:36:46,885 INFO    MainThread:4379 [wandb_init.py:init():522] starting run threads in backend
2022-02-12 20:36:48,407 INFO    MainThread:4379 [wandb_run.py:_console_start():1538] atexit reg
2022-02-12 20:36:48,408 INFO    MainThread:4379 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2022-02-12 20:36:48,409 INFO    MainThread:4379 [wandb_run.py:_redirect():1417] Redirecting console.
2022-02-12 20:36:48,411 INFO    MainThread:4379 [wandb_run.py:_redirect():1473] Redirects installed.
2022-02-12 20:36:48,411 INFO    MainThread:4379 [wandb_init.py:init():547] run started, returning control to user process
2022-02-12 20:36:48,413 INFO    MainThread:4379 [wandb_run.py:_config_callback():787] config_cb None None {'input': 'clean_telco.csv:latest', 'test_size': 0.3, 'random_seed': 40}
2022-02-12 20:38:42,167 INFO    MainThread:4379 [wandb_run.py:_atexit_cleanup():1508] got exitcode: 1
2022-02-12 20:38:42,168 INFO    MainThread:4379 [wandb_run.py:_restore():1480] restore
2022-02-12 20:38:44,718 INFO    MainThread:4379 [wandb_run.py:_wait_for_finish():1630] got exit ret: file_counts {
  wandb_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 234228
  total_bytes: 234228
  deduped_bytes: 233269
}

